src/Io/PauseBufferStream.php

```diff
@@ -29,7 +29,7 @@ class PauseBufferStream extends EventEmitter implements ReadableStreamInterface
     private $dataPaused = '';
     private $endPaused = false;
     private $closePaused = false;
-    private $errorPaused = null;
+    private $errorPaused;
```

Comment:
I understand that these assignments are essentially a NOOP. I've intentionally added these in the past to explicitly state that this property isn't initialized as part of the constructor (see also other occurrences). Would love to hear some more thoughts on this, but won't block this either way. :+1:

Reply:
As for me, I think that the constructor itself says more explicitly about it, rather then assigned `null` default value. The idea behind these assignments wasn't clear for me, so I've considered them as useless. But if it is a common practice in the code, of course I will undo this change.